libs/asset-viewer/src/lib/components/map-controls/map-controls.component.html

```diff
@@ -7,14 +7,3 @@
 <button asset-sg-icon-button [matTooltip]="'map.zoomOut' | translate" (click)="zoom.zoomOut()">
   <svg-icon key="zoom-minus" />
 </button>
-
-<span aria-hidden="true" class="divider"></span>
```

Comment:
Du to removing this, we can also get rid of the `.divider` css class as a whole.